tests/test_advanced.py

```diff
@@ -1,16 +1,98 @@
 # -*- coding: utf-8 -*-
 
 from .context import feature_mining
```

Comment:
Do you have any specific conventions you want me to follow for test cases?

Reply:
some loose recommendations, yes. let's sync on this